CONTRIBUTING.md

```diff
@@ -19,9 +23,57 @@
 
 Thank you for considering making contributions to the Interchain Security (ICS) repository! 🎉👍
 
+## Overview
+
 Contributing to this repo can mean many things such as participating in
-discussion or proposing code changes. To ensure a smooth workflow for all
-contributors, a general procedure for contributing has been established:
+discussion or proposing code changes. 
+Following the processes outlined in this document will lead to the best 
+chance of landing changes in a release.
+
+### Core team responsibility
```

Comment:
There are multiple core teams, see https://interchain.io/ecosystem
Think we should make clear the difference between core / stewarding team. Informal are stewarding ICS & core. 
Whereas Confio are core.